sdk/cosmosdb/cosmos/test/public/functional/item.spec.ts

```diff
@@ -883,3 +883,197 @@ describe("patch operations", function () {
     });
   });
 });
+
+describe("Item CRUD with priority", function (this: Suite) {
```

Comment:
In these tests we have not verified throttling based on priority, If possible can we add it?

Reply:
Currently this is not possible from SDK. We verify it on the backend if priority header is being passed.